toolbox/siesta/barriers/BarriersIO.py

```diff
@@ -70,7 +71,7 @@ def __init__(self,
         self.final_relaxed_fdf_name = final_relaxed_fdf_name
         self.neb_results_path = neb_results_path
 
-    def write_all_images_sisl(self, fname = 'images',folder_name = 'all' , out_format = 'xyz'):
+    def Write_All_Images(self, fname = 'images',folder_name = 'all' , out_format = 'xyz'):
```

Comment:
I don't particularly like CamelCase. However, it is your choice ;)
It just goes against all sisl commands. So if you want it, you should really use CamelCase everywhere.

Reply:
So i think i don't understand this camel way vs not 🐪 ...